examples/add_shapes_with_text.py

```diff
@@ -19,14 +19,14 @@
     np.array([[111, 336], [220, 336], [220, 240], [111, 240]]),
 ]
 
-# create properties
-properties = {
+# create features
+features = {
     'likelihood': [21.23423, 51.2315, 100],
     'class': ['hand', 'face', 'camera'],
 }
 edge_color_cycle = ['blue', 'magenta', 'green']
 
-text_properties = {
+text_features = {
```

Comment:
Maybe just call this `text`? These are effectively the parameters that will be passed to `TextManager`, so I think `text_parameters` could also work if that's too terse/unclear.

I think `text_features` confuses this with `Layer.features`.

Reply:
:+1: